### ISSUE TITLE
Hail plugin question

### ISSUE THREAD

**Issue by samkleeman1** [Illumina/gvcfgenotyper#15]
Hi,

Many thanks for making this available. I have used this repository to convert VCF files into a multi sample VCF using the following code:

```
ref=/mnt/grid/janowitz/home/references/human_dna/references/references/Homo_sapiens/GATK/GRCh38/Sequence/WholeGenomeFasta/Homo_sapiens_assembly38.fasta
gvcfs=/mnt/grid/janowitz/rdata_norepl/pan_immuno/germline.txt
bin=/mnt/grid/janowitz/home/applications/gvcfgenotyper/bin/gvcfgenotyper \

for i in {1..22} X;
do
    echo -r $i -f $ref -l $gvcfs -Oz -o pan_immuno_germline.chr${i}.vcf.gz;
 done | xargs -l -P 23 $bin
```

I then have tried to convert into the correct format to import into Hail using the code below. However, the file size of the original file is several GB and the file size of the output file is less than 1MB, I suspect some data is being lost/whole file is not being processed correctly.

```
export BCFTOOLS_PLUGINS=/mnt/grid/janowitz/home/applications/bcftools-1.11/plugins

for i in {1..22} X Y;
do
  /mnt/grid/janowitz/home/applications/bcftools-1.11/bin/bin/bcftools view pan_immuno_germline.chr${i}.vcf.gz | bcftools +ilmn2hail | bcftools view -Oz -o pan_immuno_germline_hail.chr${i}.vcf.gz
done
```

Really grateful for your help with this.

Kind regards,

Dr Sam Kleeman
PhD Student
Cold Spring Harbor Laboratory, NY

**Comment by samkleeman1** [comment]
To follow up on this, the reason why this is happening is that the bcftools plugin is producing a `Segmentation fault (core dumped) `error

**Comment by samkleeman1** [comment]
This is fixed if you use the older BCFtools version (1.9)
